### DISCUSSION TITLE
Avante + Vim Plug?

### DISCUSSION THREAD

**Discussion by jikkuatwork** [yetone/avante.nvim#479] [General]
Can this be installed with Vim Plug? Or are there any hard dependencies with Lazy?

**Comment by pcho** [comment]
Just below the lazy.nvim section in the README, you can find vim-plug. :-)

**Comment by jikkuatwork** [comment]
So sorry, I somehow missed it!

Anyways, I was trying to install but got stuck here:

```
Error detected while processing function <SNR>4_lod_cmd:
line    3:
E492: Not an editor command: AvanteAsk
Press ENTER or type command to continue
```

My [init.lua](https://rentry.co/init-lua)
